core/src/main/java/io/questdb/cairo/arr/ArrayTypeDriver.java

```diff
@@ -282,6 +282,7 @@ public static BorrowedArray getPlainValue(long addr, @NotNull BorrowedArray valu
         final int type = Unsafe.getUnsafe().getInt(addr);
         addr += Integer.BYTES;
         int nDims = ColumnType.decodeArrayDimensionality(type);
+        assert nDims > 0;
```

Comment:
can we move this assert inside `decodeArrayDimensionality` and to make it consistent and non-duplicated?

Reply:
i will look into this

Reply:
This assertion is present here to make sure that we don't call driver for weak dimensionality. It's not possible to move the assertion `ColumnType.decodeArrayDimensionality()` because the assertion depends on the calling code. But it's possible to remove some redundant assertions - I'll do that.